src/transformers/models/fuyu/image_processing_fuyu.py

```diff
@@ -260,6 +260,24 @@ def __init__(
         self.do_rescale = do_rescale
         self.rescale_factor = rescale_factor
         self.patch_size = patch_size if patch_size is not None else {"height": 30, "width": 30}
+        self._valid_processor_keys = [
```

Comment:
Missing validation in the preprocess call for this model?

Reply:
ah, yes, well there is no `**kwargs` in this one so no need for validation I suppose?

Reply:
Ah, yes, in that case there's no need :)